armedilla_alliyah_samantha/countries.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Countries of the World</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="container">
+    <h1>Countries of the World</h1>
+    <div class="search-container">
+      <input type="text" id="country-input" placeholder="Enter country name">
+      <button id="search-btn">Search</button>
+    </div>
+    <div id="country-details"></div>
+    <div id="region-countries"></div>
```

Comment:
If you're done changing IDs in your index.html, do not forget to change also your IDs naming in country.html, so that it will get the appropriate id. For example:

❌ id="country-input"
✅ id="country_input"

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5